terraform/digitalocean/backend/provider.tf

```diff
@@ -0,0 +1,2 @@
+provider "digitalocean" {  
+}
```

Comment:
Please, move `provider` section to `main.tf` and create `terraform.tf` file where you pin provider version

You can use [this](https://github.com/shalb/cluster.dev/blob/cb2a13efd4caa944c67d7df5b912ce025525f0ce/terraform/aws/backend/terraform.tf) as example